iot/iot-tenant-cleaner/Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM quay.io/enmasse/java-base:11-4
+FROM quay.io/ctronenmassetesting/java-base:11-5
```

Comment:
Is this a temp workaround? Or just missed?

Reply:
I need to wait for @lulf to push the images.

Reply:
Updated.